src/sentry/migrations/0353_fix_org_slug_casing.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 2.2.28 on 2023-01-31 20:37
+
+from django.db import migrations
+from django.db.models.functions import Lower
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def fix_org_slug_casing(apps, schema_editor):
+    Organization = apps.get_model("sentry", "Organization")
+    query = Organization.objects.exclude(slug=Lower("slug"))
```

Comment:
In my testing this does a seq scan. I don't really think we can use an index here. However even so, the query returned pretty fast, so it's probably safe anyway, since realistically it'll only make a single query